src/reason-parser/reason_pprint_ast.ml

```diff
@@ -2968,7 +2975,7 @@ let printer = object(self:'self)
             match rf with
               | Rtag (label, attrs, opt_ampersand, ctl) ->
                 let pcd_name = {
-                  txt = label;
+                  txt = label.txt;
                   loc = pcd_loc;
```

Comment:
This should probably become `label.loc`? (And finally fixes the comment duplication?)